### ISSUE TITLE
Modify table statement uses double quotes

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/sqlitebrowser#683]
I normally use DB4S to create my databases, and when you create a new table, DB4S uses grave accents around each identifier.

![image](https://cloud.githubusercontent.com/assets/7872218/17189392/1f0813aa-5439-11e6-89d9-7a3b77a7278e.png)

This is also reflected in the schema column.
However, as soon as you modify the table, the schema column changes to double quotes.

![image](https://cloud.githubusercontent.com/assets/7872218/17189451/67001766-5439-11e6-9ef7-f79c20e3c2bb.png)

However, the 'create table' window at the bottom of the 'Modify Table' dialog still uses grave accents.

I remember reading recently about double quotes and apostrophes (single quotes), but can't find it now.  But I did see this about quoting, and it said using grave accents was purely for MySQL backwards compatibility.
https://www.sqlite.org/lang_keywords.html

It suggested using double quotes for identifiers.   I note the examples on the sqlite website doesn't use any quotes at all, apart from when to escape an apostrophe:
`INSERT INTO xyz VALUES('5 O''clock');`
Again, suggesting strings should be enclosed in single quotes rather than grave accents.

Should a standard be adopted, rather than a mix of bits?

**Comment by justinclift** [comment]
Yeah, being consistent in our approach is probably a good to make happen. :smile:

**Comment by revolter** [comment]
But how do we decide wether to use ` or "? Or do we make yet another user preference setting for it?

**Comment by justinclift** [comment]
In theory :smile:, that shouldn't be ***too hard*** to figure out:

1. If the SQLite approach is documented well enough to figure it out, use that
2. If there are spots that aren't completely clear from that, lets see if we can figure it out from the SQL standards
3. If that doesn't work then copy PostgreSQL's approach.  aka [What Would Postgres Do?](https://youtu.be/ZvmMzI0X7fE?t=3m8s)

**Comment by chrisjlocke** [comment]
>But how do we decide whether to use \` or "? Or do we make yet another user preference setting for it?
From my understanding, there is no reason to use grave accents (\`)
They are there solely for mySql compatability.  So we should be using double quotes and single quotes only.  No preferences needed.

`create table "blobby" ("field1" text, "field2" text);`
`insert into "blobby" ("field1", "field2") values ('my text values','my text values')`

**Comment by justinclift** [comment]
This is related to #629 too.

**Comment by revolter** [comment]
So an user preference setting as I said? :smile: So we don't bump into other users who want compatibility with MySQL.

**Comment by justinclift** [comment]
Looks like it? :wink:

**Comment by revolter** [comment]
I guess so

**Comment by revolter** [comment]
We could default it to quotes, and add the option to change it in an "Advanced" section, as to not clutter it (like in @chrisjlocke's SS :laughing:). And probably move the "escape character" and/or other settings in the "Advanced" section too while we're at it.

**Comment by steronydh** [comment]
Double quotes would be good!

**Comment by mgrojo** [comment]
@chrisjlocke The configuration for identifier quotes is already in the nightly build. Would you have some time for testing it?
